### PR TITLE
Auto unchecked accepted testcase

### LINKED ISSUE #734: Auto-uncheck accepted test cases
**Is your feature request related to a problem? Please describe.**

Now we need to manually click "Uncheck Accepted" in the "More" menu of the test cases. Some users may want to automate this process.

**Describe the solution you'd like**

Add an option to auto-uncheck accepted test cases.

### PR DESCRIPTION
**Pull request by neko-para**
## Description
Allow editor auto uncheck the accepted testcases.
Add an option for it.
Add Chinese translation.

## Related Issues / Pull Requests
Closes #734

## Checklist

- [x] If the key of a setting is changed, the `old` attribute is updated or it is resolved in SettingsUpdater.
- [x] If there are changes of the text displayed in the UI, they are wrapped in `tr()` or `QCoreApplication::translate()`.
- [ ] If needed, I have opened a pull request or an issue to update the [documentation](https://github.com/cpeditor/cpeditor.github.io).
- [x] If these changes are notable, they are documented in [CHANGELOG.md](https://github.com/cpeditor/cpeditor/blob/master/CHANGELOG.md).

### CODE REVIEW

File: src/Settings/PreferencesWindow.cpp [hunk 1]
```diff
@@ -244,7 +244,7 @@ PreferencesWindow::PreferencesWindow(QWidget *parent) : QMainWindow(parent)
 #endif
             .page(TRKEY("Save Session"), {"Hot Exit/Enable", "Hot Exit/Auto Save", "Hot Exit/Auto Save Interval"})
             .page(TRKEY("Bind file and problem"), {"Restore Old Problem Url", "Open Old File For Old Problem Url"})
-            .page(TRKEY("Empty Test Cases"), {"Run On Empty Testcase", "Check On Testcases With Empty Output"})
+			.page(TRKEY("Test Cases"), {"Run On Empty Testcase", "Check On Testcases With Empty Output", "Auto Uncheck Accepted Testcases"})
```
Comment (sonulohani) [inline src/Settings/PreferencesWindow.cpp:247]: Remove extra spaces

**Review by sonulohani — CHANGES_REQUESTED** [review]

**Review by sonulohani — APPROVED** [review]
LGTM.

**Review by sonulohani — CHANGES_REQUESTED** [review]
LGTM

**Review by sonulohani — DISMISSED** [review]

File: src/Settings/settings.json [hunk 2]
```diff
@@ -1239,5 +1239,11 @@
         "name": "Promotion Dialog Shown",
         "type": "bool",
         "notr": true
+    },
+    {
+        "name": "Auto Uncheck Accepted Testcases",
+        "desc": "Auto uncheck accepted testcases",
+        "type": "bool",
+        "default": "false"
```
Comment (coder3101) [inline src/Settings/settings.json:1247]: Formatting issue:

You can make sure it's correctly formatted it in two ways:

1. Copy paste an already formatted setting and edit it (Recommended)
2. Format using vscode default JSON formatter.

**Review by coder3101 — COMMENTED** [review]
After this PR it makes sense to have "Run on all test cases before submit"

**Review by coder3101 — APPROVED** [review]

**Review by coder3101 — APPROVED** [review]

**Comment by coder3101** [comment]
#735 priority increased because of this feature. Before submitting all testcases should be ran.

**Comment by ouuan** [comment]
We need to update the docs.

**Comment by neko-para** [comment]
The document seems to be updated.
[#152](https://github.com/cpeditor/cpeditor.github.io/pull/152)

**Comment by ouuan** [comment]
> The document seems to be updated.
> [#152](https://github.com/cpeditor/cpeditor.github.io/pull/152)

Sorry, I checked the commits and forgot to check the PRs.

**Pull request merged**